.eslintrc.js

```diff
@@ -23,6 +23,10 @@ module.exports = {
       },
     },
   },
+  parserOptions: {
+    project: './tsconfig.json',
+    tsconfigRootDir: __dirname,
+  },
```

Comment:
This might've not been caused by the inclusion of `styled-components` but it was an error thrown by eslint so lemme fix